### ISSUE TITLE
Refs to the theme included .js and .css files should respect baseURL

### ISSUE THREAD

**Issue by tattwamasi** [jpescador/hugo-future-imperfect#22]
The templates use hard coded relative references for the files.  This breaks when deploying to anything that isn't a host only URL, i.e,  deploying to http://blog.joe.com works, but http://joe.com/blog would not, because it would look for the .js and .css under http://joe.com/js and http://joe.com/css rather than http://joe.com/blog/js etc.

I think changing the links in the templates to
`href="{{ .Site.BaseUrl }}/css/whatever.css"` should work.

Will try and see.  I assume the extra slashes get stripped.

**Comment by tattwamasi** [comment]
Well, I was close.  I think this handles it, for example:
```
<script src={{"/js/jquery.min.js" | relURL }}></script>
```
And
```
<link rel="stylesheet" href={{ "/css/main.css" | relURL }} />
```

There's a similar issue for the config.toml entry of params.intro.pic.src but not sure yet how to fix that, since the user could stick a relative or absolute URL in there, and can't use the {{ }} things themselves in the config file (right?).

**Comment by tattwamasi** [comment]
In sidebar.html you can fix the params.intro.pic.src issue by doing something like:
`<img src="{{ relURL . }}" class="intro-circle" width="{{ $pic.width }}" alt="{{ $pic.alt }}" />`

But at this point I'm seeing the same basic issue everywhere.  The post button, category links, etc. on the side bar, etc. 

It all comes from the situations where relative URLs are hardcoded into the theme files.  All of them should be processed as above in order to make them where in situations  where the site is not at the root of a host.

This situation arises for GitLab and GitHub project sites, for example.

I'll start working on it.  I've made some other tree changes in my forked version, so not promising a pull request, but you can definitely look at what I've done.  I've only just started looking at Hugo, so you or one of the other users may see a different and better way to fix this.

**Comment by pacollins** [comment]
Was being covered by #27. We need to revisit with a fresh PR.

**Comment by pacollins** [comment]
Solved in #48
